src/taipy/core/scenario/scenario.py

```diff
@@ -135,19 +135,26 @@ def pipelines(self, pipelines: Union[List[PipelineId], List[Pipeline]]):
 
     @property
     def tasks(self) -> Dict[str, Task]:
```

Comment:
```suggestion
    def tasks(self) -> Dict[str, List[Task]]:
```
Since there is no auto reload on the `tasks` property, we can just remove _get_tasks. No?

Reply:
I think for consistency, why not keep it?